### PR TITLE
[RFR] Optimized travis

### PR DESCRIPTION
**Pull request by quarckster**
Removed bash from the .travis.yml. ~~Firstly I tried to remove it completely, but [webdriverdownloader ](https://pypi.org/project/webdriverdownloader/) fails to download the geckodriver on some builds.~~ Besides additional stage `deploy` has been included. This stage starts only when a commit gets a tag.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
+  include:
+    - stage: deploy
+      python: 3.6
+      before_install:
+      after_success:
+      install: pip install -U setuptools setuptools_scm
+      script: skip
+      deploy:
+        provider: pypi
+        user: mfalesni
```
Comment (mshriver) [inline .travis.yml]: We can't auto-deploy using mfalesni's account anymore, but can leave this in for now. Do you want to own auto-deploy for wt.core?

File: .travis.yml [hunk 2]
```diff
@@ -1,39 +1,43 @@
 language: python
+addons:
+  firefox: '61.0.2'
```
Comment (mshriver) [inline .travis.yml]: Why this version? I'm thinking we use the `latest-esr` alias here to stay on ESR.

File: .travis.yml (hunk 1 above)
Comment (quarckster) [reply .travis.yml]: Sure, no problem

File: .travis.yml (hunk 2 above)
Comment (quarckster) [reply .travis.yml]: I just took the same version as it was [before](https://github.com/RedHatQE/widgetastic.core/pull/118/files#diff-354f30a63fb0907d4ad57269548329e3L25), but of course `latest-esr` makes more sense.

**Pull request merged**